models/job.go

```diff
@@ -26,15 +31,22 @@ type JobLinks struct {
 }
 
 // NewJob returns a new Job resource that it creates and populates with default values.
-func NewJob(id string) Job {
+func NewJob(id string) (Job, error) {
 	zeroTime := time.Time{}.UTC()
+	cfg, err := config.Get()
+	if err != nil {
+		return Job{}, fmt.Errorf("%s: %w", errors.New("unable to retrieve service configuration"), err)
+	}
+	urlBuilder := url.NewBuilder("http://" + cfg.BindAddr)
```

Comment:
It might be worth specifying the expected format for `BindAddr` in the readme  i.e. it doesn't require a scheme.

Reply:
done